mapalign/embed.py

```diff
@@ -77,32 +81,8 @@ def compute_diffusion_map(L, alpha=0.5, n_components=None, diffusion_time=0,
             raise ImportError('Checks require scikit-learn, but not found')
 
     ndim = L.shape[0]
-    if overwrite:
```

Comment:
`overwrite` is no longer supported in the rewrite. i think this should still be used as a memory saving option.

Reply:
@mfalkiewicz - this is not supported in the refactor